app/Http/Requests/Legacy/Two/PostRequest.php

```diff
@@ -31,7 +31,7 @@ public function rules()
             'caption' => 'nullable|string',
             'status' => 'in:pending,accepted,rejected',
             'source' => 'nullable|string',
-            'file' => 'image|dimensions:min_width=400,min_height=400,max_width=5000,max_height_4000',
+            'file' => 'dimensions:min_width=400,min_height=400,max_width=5000,max_height_4000',
```

Comment:
I think the `dimensions` rule requires a multi-part file upload as well. Can you double-check this works?

Reply:
@DFurnes yes - just double checked! :) 
![screen shot 2018-05-31 at 2 02 56 pm](https://user-images.githubusercontent.com/9019452/40799356-60dc061a-64db-11e8-990c-6e451775f3d0.png)

Reply:
ah you're right @DFurnes - it gives the same error for a string with valid dimensions :-/ Should I just remove this? I think that we should be ok to wait til Phoenix moves to the v3 endpoints since it was a one time occurrence?

Reply:
Yeah, I think that's fine. 👍